### PR TITLE
feat: Added support to sidebar

### LINKED ISSUE #2095: Add Discord + Support to the left navigation
Add the request support and a link to Discord, from the command + K to the sidebar. 

People are less likely to find that hidden feature that isn't advertised anywhere causing friction to find support or our discord.

### PR DESCRIPTION
**Pull request by harshsbhat**
## What does this PR do?

Fixes #2095

Adds the support options to side bar

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [Contributing Guide](./CONTRIBUTING.md)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand areas
- [x] Ran `pnpm build`
- [x] Ran `pnpm fmt`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

## Summary by CodeRabbit

- **New Features**
	- Introduced a Discord navigation item in the sidebar for easy access.
	- Added a Feedback dialog component for users to report issues directly from the dashboard.
- **Enhancements**
	- Expanded the types of icons that can be used in the sidebar navigation.
	- Streamlined the feedback process by integrating feedback directly into the command menu.
	- Simplified feedback submission with a new FeedbackCommand component.
	- Improved user experience with toast notifications for feedback submission outcomes.

### CODE REVIEW

**Comment by harshsbhat** [comment]
I have created a new component for the sidebar because it was not allowing me to use `CommandItem` outside the `Commands`. Or even use the `div` in `CommandGroup`

File: apps/dashboard/components/dashboard/feedback-component.tsx
Comment (chronark) [inline apps/dashboard/components/dashboard/feedback-component.tsx:1]: This is duplicated now, right?
can we also use this component in the cmd-k menu?

**Review by chronark — CHANGES_REQUESTED** [review]
CleanShot 2024-09-23 at 10 25 04@2x
The docs look weird now cause it's no longer a single icon that tells the user it'll open a new tab.

Let's use `Book` or `BookOpen` instead to blend in

File: apps/dashboard/components/dashboard/feedback-component.tsx
Comment (harshsbhat) [reply apps/dashboard/components/dashboard/feedback-component.tsx:1]: sure. on it

**Comment by harshsbhat** [comment]
@chronark I have pushed the requested changes for this one too

**Comment by chronark** [comment]
CleanShot 2024-09-26 at 13 19 12@2x
can you check why the height isn't consistent?

**Comment by harshsbhat** [comment]
@chronark 

The length is now consistent. but the problem is 

image

same with command

**Comment by chronark** [comment]
why not?
It's clickable in prod, where we only have the docs link right now, so this sounds like a regression within this PR

**Comment by harshsbhat** [comment]
Yeah I know it's working in prod. I just wanted to update you that I am still working on it. So you don't accidentally keep cloning my branch it just to realise this error.

**Comment by harshsbhat** [comment]
It should be good for review now. The heights and lengths are consistent. It also works with arrow keys for the command menu again.

**Comment by harshsbhat** [comment]
My bad. still not ready for review

**Comment by harshsbhat** [comment]
Done. I think this is ready to review now. I have made all heights and widths consistent.

**Comment by harshsbhat** [comment]
@chronark 

You will probably need to submit the form to check if it is working I was not able to do so. As I don't have a Plain API key. It should work tbh. As I haven't made any other changes in the code other than styling

**Review by chronark — APPROVED** [review]

**Pull request merged**